### ISSUE TITLE
GreaterThanOrEqual with multiplier

### ISSUE THREAD

**Issue by lipka** [nakiostudio/EasyPeasy#28]
First let me start off by saying EasyPeasy is easily my go to auto layout wrapper. It's really awesome! 💪

However one thing I found missing is to facilitate the creation of constraints that would allow for specifying both the relation and the multiplier at the same time.

Example:
```
NSLayoutConstraint(item: view, attribute: .Width, relatedBy: .GreaterThanOrEqual, toItem: otherView, attribute: .Width, multiplier: 0.5, constant: 1)
```

I think this is currently not possible with any of the operators and also the way the `Constant`/`Modifier` and `layoutRelation`/`layoutMultiplier` are set up.

Thanks!

**Comment by nakiostudio** [comment]
Hi @lipka 👋 

Thanks for those words, I really appreciate it!

Regarding your comment, you are absolutely right. That's something that I decided to sacrifice in the beginning of the project just for simplicity. I will think about it 👍

**Comment by nakiostudio** [comment]
I'm working on it @lipka 👍

**Comment by nakiostudio** [comment]
Hi again @lipka 👋 

I've released `1.1.0` that includes your feature request. It'd be great to know your thoughts about it!

***

## v.1.1.0

* Now it's possible to combine `multipliers` with `Equal`, `GreaterThatOrEqual`
and `LessThanOrEqual` relations.

```swift
// i.e.
Width(>=20*0.5) // value = 20, multiplier = 0.5, relation = .GreaterThanOrEqual
Width(<=20*0.5) // value = 20, multiplier = 0.5, relation = .LessThanOrEqual
Width(==20*0.5) // value = 20, multiplier = 0.5, relation = .Equal
```
